views/templates/admin/configuration.tpl

```diff
@@ -68,8 +69,7 @@
     <div class="mj_landing">
         <div align="center">
             <br />
-            <a href="https://www.mailjet.com/support"
-               target="_blank">{l s='If you have a question or if you have a problem, click here to contact support.' mod='mailjet'}</a>
+            {{l s='If you have a question or if you have a problem, click here to contact support.' mod='mailjet'}|html_entity_decode}
```

Comment:
Hi @yasen-mailjet,

Sorry for this late answer. If you really want to add HTML code in your translations, here is an example you can reuse: https://github.com/PrestaShop/PrestaShop/blob/6c9365135d455938a266461728b346ed43b54b2f/admin-dev/themes/default/template/controllers/backup/helpers/list/list_header.tpl#L71
But in this case, you cannot have a URL for each language.